xcm/src/v3/junctions.rs

```diff
@@ -29,7 +29,9 @@ pub(crate) const MAX_JUNCTIONS: usize = 8;
 ///
 /// Parent junctions cannot be constructed with this type. Refer to `MultiLocation` for
 /// instructions on constructing parent junctions.
-#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug, TypeInfo, MaxEncodedLen)]
+#[derive(
+	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug, TypeInfo, MaxEncodedLen,
+)]
 pub enum Junctions {
```

Comment:
I must have missed this in the previous PRs, but since `Junctions` are now `Copy`able, a lot of methods can in fact be `const` nowadays, the most important of which is `new`. This will allow `MultiLocation`s to be constructed in a const context, and one of the more interesting things that we can do is to statically assert that every `MultiLocation` constructed by the developer is at its most simplified form, i.e. they do not contain the `Ancestry` as a prefix.

Reply:
Nice yeah. I wasn't planning on necessarily leaving the `Copy` trait in, but for now it's required due to `MultiLocation` being used as an `Instance` type in Uniques pallet config.